src/NFun.Fuspec.TestHandler/TestResultConverter.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using Nfun.Fuspec.Parser.Model;
+using NFun.Fuspec.TestHandler.Models;
+
+namespace NFun.Fuspec.TestHandler
+{
+    public static class TestResultConverter
+    {
+        public static ExceptionTestCaseResult BadTestResult(long ms, FuspecTestCase testCase, Exception e) 
```

Comment:
Do not use postfix in such situation. just place factories into parent class to use it like this:

TestCaseResult.Bad(testCase, buildTime, e)
TestCaseResult.ParseFailure(testCase, buildTime, e)
TestCaseResut.Passed(testCase, buildTime)